README.md

```diff
@@ -1,11 +1,11 @@
 [![Build Status](https://travis-ci.org/dashaluna/hm-content-toc.svg?branch=master)](https://travis-ci.org/dashaluna/hm-content-toc)
 
 # HM Content TOC #
-**Contributors:** dashaluna, tcrsavage, johnbillion, sanchothefat  
+**Contributors:** dashaluna, tcrsavage, johnbillion, sanchothefat, humanmade  
 **Tags:** TOC, TOC shortcode, shortcode, content TOC, post TOC, page TOC, TOC for content, table of content, table of contents, HM, Human Made  
 **Requires at least:** 4.2  
 **Tested up to:** 4.3  
-**Stable tag:** 1.0.1
+**Stable tag:** 1.0.1  
```

Comment:
@johnbillion Do I change the version before the release of that version or when I start work on something new? Does it make sense?